corehq/apps/repeaters/models.py

```diff
@@ -334,7 +376,10 @@ def get_url(self, repeat_record):
             # adapted from http://stackoverflow.com/a/2506477/10840
             url_parts = list(urlparse.urlparse(url))
             query = urlparse.parse_qsl(url_parts[4])
-            query.append(("app_id", self.payload_doc(repeat_record).app_id))
+            try:
+                query.append(("app_id", self.payload_doc(repeat_record).app_id))
+            except (XFormNotFound, ResourceNotFound):
+                return None
```

Comment:
Sounds like the current behaviour is incorrect? If I call `get_url` and it returns `None`, what will the RepeatRecord do when it fires?